index.bs

```diff
@@ -291,7 +280,7 @@ interface FileSystemDirectoryHandle : FileSystemHandle {
 
   Promise<sequence<USVString>?> resolve(FileSystemHandle handle);
 
-  Promise<void> removeRecursively();
+  Promise<void> removeEntry(USVString name, optional FileSystemRemoveOptions options);

```

Comment:
While I personally like `remove`, the web platform seems to have settled on `delete` for this operation.

So, WDYT of `delete`, or `deleteEntry`?

Also fine to leave the name as-is and let the TAG decide when we submit a review.

Reply:
I think I'll leave as is for now. Which web platform APIs were you referring to btw? I guess CacheStorage indeed uses `delete`, but for example DOM has `removeChild` and `removeAttribute`